examples/cpp/modes/mode_with_executor/include/mode.hpp

```diff
@@ -47,7 +47,10 @@ class FlightModeTest : public px4_ros2::ModeBase
 
     const float elapsed_s = (now - _activation_time).seconds();
     const Eigen::Vector3f velocity{10.f, elapsed_s * 2.f, -2.f};
-    _trajectory_setpoint->update(velocity);
+    const Eigen::Vector3f acceleration{NAN, NAN, NAN};
+    const float yaw = NAN;
+    const float yaw_rate = NAN;
```

Comment:
You can remove those now and use the defaults (below too)

Reply:
ah good point, thanks